libbuteosyncfw/profile/ProfileManager.cpp

```diff
@@ -140,7 +137,7 @@ Profile *ProfileManagerPrivate::load(const QString &aName, const QString &aType)
 
 SyncLog *ProfileManagerPrivate::loadLog(const QString &aProfileName)
 {
-    QString fileName = iPrimaryPath + QDir::separator() + Profile::TYPE_SYNC + QDir::separator() +
+    QString fileName = iConfigPath + QDir::separator() + Profile::TYPE_SYNC + QDir::separator() +
```

Comment:
Was first thinking for splitting the log path but ended up keeping them together as simpler. Then again since there are these log classes to get the history, are these really cache either? Think commonly the cache should be possible to remove without side-effects except just the time to generate or download content again. If the history is removed, it's gone.

Reply:
The history thing is a good point. In my opinion, nothing should rely on the history. It's too fragile, the file could be removed without notice. I remembered that CalDAV a long time ago relied on the hisory to get the last sync date time. If the log disappeared or get jammed by "error no network" (the log is hard coded limited to the latest 5 entries), the last sync date time was not available anymore. That's why I'm tempted to see the history as a disposable information suited for cache.

But anyway, the PR is already very good like that to good in.